packages/server/src/threads/automation.ts

```diff
@@ -742,6 +756,7 @@ export function execute(job: Job<AutomationData>, callback: WorkerCallback) {
     workspaceId,
     automationId,
     task: async () => {
+      await reloadAutomation(job)
```

Comment:
I might be wrong, but I think this is not used at all. Might it be that we are only using the `executeInThread`?